<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>840</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="tileEditorContainer" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>256</width>
      <height>544</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>544</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>544</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="mapEditorContainer" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>800</width>
      <height>608</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>608</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <widget class="QToolButton" name="plusButtonEast">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>190</y>
       <width>22</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/icons/icone-plus.png</normaloff>../resources/icons/icone-plus.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="minusButtonEast">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>230</y>
       <width>22</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/icons/icone-moins.png</normaloff>../resources/icons/icone-moins.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listTileSet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TileSetListLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste des TileSets</string>
    </property>
   </widget>
   <widget class="QLabel" name="tileSetNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TilesetName</string>
    </property>
   </widget>
   <widget class="QLabel" name="cooLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[  0    :   0    ]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionClose_Project"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionCollision_Mode"/>
    <addaction name="actionPen_mode"/>
    <addaction name="actionRectangle_mode"/>
    <addaction name="actionFill_mode"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDatabase"/>
    <addaction name="actionEvent_mode"/>
    <addaction name="actionResource_manager"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuMap"/>
   <addaction name="menuTools"/>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
  </widget>
  <widget class="QToolBar" name="secondToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_Project"/>
   <addaction name="actionClose_Project"/>
   <addaction name="actionCollision_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionDatabase"/>
   <addaction name="actionEvent_mode"/>
   <addaction name="actionResource_manager"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QToolBar" name="mapToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionPen_mode"/>
   <addaction name="actionRectangle_mode"/>
   <addaction name="actionFill_mode"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/folder_add_16.png</normaloff>../resources/icons/folder_add_16.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/open_16.png</normaloff>../resources/icons/open_16.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/save_16.png</normaloff>../resources/icons/save_16.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/folder_delete_16.png</normaloff>../resources/icons/folder_delete_16.png</iconset>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionCollision_Mode">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/etape1_16.png</normaloff>../resources/icons/etape1_16.png</iconset>
   </property>
   <property name="text">
    <string>Lower mode</string>
   </property>
  </action>
  <action name="actionPen_mode">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/brush_16.png</normaloff>../resources/icons/brush_16.png</iconset>
   </property>
   <property name="text">
    <string>Pen mode</string>
   </property>
  </action>
  <action name="actionRectangle_mode">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/rectangle_16.png</normaloff>../resources/icons/rectangle_16.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle mode</string>
   </property>
  </action>
  <action name="actionFill_mode">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/color-fill_16.png</normaloff>../resources/icons/color-fill_16.png</iconset>
   </property>
   <property name="text">
    <string>Fill mode</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/database_16.png</normaloff>../resources/icons/database_16.png</iconset>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionEvent_mode">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/event_16.png</normaloff>../resources/icons/event_16.png</iconset>
   </property>
   <property name="text">
    <string>Event mode</string>
   </property>
  </action>
  <action name="actionResource_manager">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/resource_manager_16.png</normaloff>../resources/icons/resource_manager_16.png</iconset>
   </property>
   <property name="text">
    <string>Resource manager</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/play_16.png</normaloff>../resources/icons/play_16.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/help_16.png</normaloff>../resources/icons/help_16.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
